Resources/Prototypes/Entities/Clothing/Ears/headsets.yml

```diff
@@ -288,3 +288,21 @@
       sprite: Clothing/Ears/Headsets/freelance.rsi
     - type: Clothing
       sprite: Clothing/Ears/Headsets/freelance.rsi
+
+- type: entity
+  parent: [ClothingHeadset, BaseCommandContraband]
```

Comment:
```suggestion
  parent: [ClothingHeadset, BaseRestrictedContraband]
```
The warden is not command, so it being command restriced shows up as contraband for them
![grafik](https://github.com/user-attachments/assets/eef4956a-47a2-42e1-b589-26332a8bee13)
For the same reason they should probably not have a command encryption key either.